### ISSUE TITLE
Unable to cast object of type 'System.Management.Automation.Language.ConstantExpressionAst' to type 'System.Management.Automation.Language.StringConstantExpressionAst'

### ISSUE THREAD

**Issue by bbollard** [adamdriscoll/poshtools#462]
Hello, using tools version 3.0.399. When I attempt to debug my script by invoking 'Execute as Script with Parameters' I receive the following exception:

"Unable to cast object of type 'System.Management.Automation.Language.ConstantExpressionAst' to type 'System.Management.Automation.Language.StringConstantExpressionAst'"

The offending parameter appears to be : 

```
      [Parameter(Mandatory=$true)]
      **[ValidateSet( 128, 512, 1023 )]**
      [int]$dataDiskSizeGB,
```
##

**Comment by adamdriscoll** [comment]
https://bitbucket.org/ironmansoftware/poshtools/issues/9/unable-to-cast-object-of-type

**Comment by adamdriscoll** [comment]
Resolved in 4.0.7
